From 707d4500880cc3e657a6fa0c76aed2c710c0158e Mon Sep 17 00:00:00 2001
From: Devaraj Ranganna <devaraj.ranganna@arm.com>
Date: Tue, 21 May 2024 11:44:48 +0000
Subject: [PATCH] Increase keep-alive interval

The MQTT server resets connection if the device doesn't send keep alive
packets within keep-alive interval, which is configured to 5 seconds by
default. Increase the keep-alive interval to 60 seconds to avoid connection
reset request when the device is busy.

Signed-off-by: Devaraj Ranganna <devaraj.ranganna@arm.com>
---
 src/mqtt/mqtt_test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/mqtt/mqtt_test.c b/src/mqtt/mqtt_test.c
index eadc176..14d30c5 100644
--- a/src/mqtt/mqtt_test.c
+++ b/src/mqtt/mqtt_test.c
@@ -170,7 +170,7 @@
  * @brief Time interval in seconds at which an MQTT PINGREQ need to be sent to
  * broker.
  */
-#define MQTT_KEEP_ALIVE_INTERVAL_SECONDS    ( 5U )
+#define MQTT_KEEP_ALIVE_INTERVAL_SECONDS    ( 60U )

 /**
  * @brief The MQTT message published in this example.
--
2.43.2

